### PR TITLE
Fix/management service

### PR DESCRIPTION
**Pull request by fedinskiy**
### Summary
1. Create a service for an application, deployed by Quarkus extension, needed due to quarkusio/quarkus#34645, required for https://issues.redhat.com/browse/QQE-401
2. Add an option to disable automatic test reruns,  #1040

Please check the relevant options

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Dependency update
- [ ] Refactoring
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] This change requires a documentation update
- [x] This change requires execution against OCP (use `run tests` phrase in comment)

### Checklist:
- [x] Example scenarios has been updated / added
- [x] Methods and classes used in PR scenarios are meaningful
- [x] Commits are well encapsulated and follow [the best practices](https://cbea.ms/git-commit/)

### CODE REVIEW

**Comment by fedinskiy** [comment]
run tests

File: pom.xml [hunk 1]
```diff
@@ -226,7 +227,7 @@
                     <configuration>
                         <!-- fixes unavailable native binary build logs, https://github.com/quarkus-qe/quarkus-test-framework/issues/785 -->
                         <forkNode implementation="org.apache.maven.plugin.surefire.extensions.SurefireForkNodeFactory" />
-                        <rerunFailingTestsCount>2</rerunFailingTestsCount>
+                        <rerunFailingTestsCount>${reruns}</rerunFailingTestsCount>
```
Comment (michalvavrik) [inline pom.xml:230]: Based on this https://maven.apache.org/surefire/maven-surefire-plugin/examples/rerun-failing-tests.html#rerun-failing-tests I expected to be able to override reruns with ` -Dfailsafe.rerunFailingTestsCount=0`, but I'm fine with this change, let's keep it

**Review by michalvavrik — APPROVED** [review]

**Comment by michalvavrik** [comment]
forgot to comment - changes LGTM, PR name not ideal :-), thanks

File: pom.xml (hunk 1 above)
Comment (fedinskiy) [reply pom.xml:230]: I also expected that, but that doesn't work out (at least for TS)

Comment (michalvavrik) [reply pom.xml:230]: ok thanks, I've never tried it

**Pull request merged**